src/webapp/pages/settings/Filter.tsx

```diff
@@ -0,0 +1,27 @@
+import { displayName, MetadataModel } from "../../../domain/entities/MetadataItem";
+import Dropdown, { DropdownOption } from "../../components/dropdown/Dropdown";
+import _ from "lodash";
+import i18n from "../../../locales";
+import { useSettingsPage } from "./useSettingsPage";
+
+export const Filter = () => {
+    const { listOptions, onChangeModel } = useSettingsPage();
```

Comment:
If you are using here this hook together with the other in SettingsPage component, then you are using two different hooks with its own functions and data. This in Filter, only changes the dropdown filter. This is the reason why the table elements are not updated when you use these filters, because only the data of this hook is being updated, and not deta from the hook in SettingsPage. To solve this, get listOptions and onChangeModel as props here, and get these from the useSettingsPage hook declared in SettingsPage component